relstorage/storage.py

```diff
@@ -1073,6 +1073,9 @@ def _abort(self):
         if self.blobhelper is not None:
             self.blobhelper.abort()
 
+    def afterCompletion(self):
+        self._rollback_load_connection()
```

Comment:
I wonder if we should also roll back the store connection, just to be on the safe side.